### DISCUSSION TITLE
Image pre-processing

### DISCUSSION THREAD

**Discussion by veenstje** [angelolab/ark-analysis#487] [Q&A] [answered]
Hello, is the lab currently using the MIBI_GUI to assist with image pre-processing (e.g., background removal, denoising, and aggregate removal) or a different method?

**Comment by ngreenwald** [comment]
We're currently using the AMP package for preprocessing: https://github.com/angelolab/amp. 

However, we're working on a new solution that we should have available sometime early in the new year that requires much less manual fine tuning.

**Reply by veenstje** [reply]
Thanks for the info. It appears this is only supported for macOS. Is there a similar application for windows available? If not, will the newer solution you mentioned be compatible with windows?

**Reply by ngreenwald** [reply]
IonPath has an option for background subtraction/denoising that works on Windows, but ours does not. The new version will only require a jupyter notebook environment and will work across platforms.

**Reply by veenstje** [reply]
Ok, I will be eagerly waiting for the new version! Thanks for your help

**Comment by ngreenwald** [comment, marked as answer]
The new preprocessing pipeline is now live at www.github.com/angelolab/toffy
